modules/localgov_campaigns_paragraphs/tests/src/Functional/CampaignsParagraphsAdministrationTest.php

```diff
@@ -94,6 +94,17 @@ public function testCampaignParagraphsTypes() {
     $this->assertSession()->pageTextContains('localgov_heading_level');
     $this->assertSession()->pageTextContains('localgov_text');
 
+    // Check 'Media with text' fields.
+    $this->drupalGet('/admin/structure/paragraphs_type/localgov_media_with_text/fields');
+    $this->assertSession()->pageTextContains('localgov_title');
+    $this->assertSession()->pageTextContains('localgov_heading_level');
+    $this->assertSession()->pageTextContains('localgov_text');
+    $this->assertSession()->pageTextContains('localgov_link');
+    $this->assertSession()->pageTextContains('localgov_media_item');
+    $this->assertSession()->pageTextContains('localgov_media_position');
+    $this->assertSession()->pageTextContains('localgov_media_with_text_style');
+    $this->assertSession()->pageTextContains('localgov_localgov_opens_in_a_new_window');
```

Comment:
Should be `localgov_opens_in_a_new_window'

Reply:
Fixed